requirements/edx-sandbox/py35.txt

```diff
@@ -4,30 +4,30 @@
 #
 #    make upgrade
 #
-common/lib/sandbox-packages
-common/lib/symmath
-git+https://github.com/edx/openedx-calc.git@e9b698c85ad1152002bc0868f475f153dce88952#egg=calc==0.4
-cffi==1.14.0
-git+https://github.com/edx/openedx-chem.git@ff4e3a03d3c7610e47a9af08eb648d8aabe2eb18#egg=chem==1.0.0
-cryptography==2.8
+common/lib/sandbox-packages  # via -r requirements/edx-sandbox/py35.in (line 24)
```

Comment:
pip-tools 4.5.1 is in the process of being released, and will remove the line numbers from these comments.  You should probably wait to rebase until that's been released and we've upgraded to it on edx-platform master.

Reply:
Master has now been updated to use pip-tools 4.5.1, and the line numbers are gone: https://github.com/edx/edx-platform/pull/23198 .  You should get a much smaller diff now if you rebase and run `make upgrade` again.